codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/SymbolVisitor.java

```diff
@@ -228,15 +235,39 @@ public Symbol memberShape(MemberShape shape) {
 
     @Override
     public Symbol timestampShape(TimestampShape shape) {
-        // TODO: add once dependency support is in
-        return createSymbolBuilder(shape, "nil").build();
+        return createSymbolBuilder(shape, "time.Time")
+                .addReference(createNamespaceReference(GoDependency.TIME))
+                .build();
+    }
+
+    private Symbol.Builder createSymbolBuilder(Shape shape, String typeName, boolean pointable) {
```

Comment:
I think this would be cleaner as `createPointableSymbolBuilder(Shape shape, String typeName)` (overloaded with + `String namespace`) that sets the pointable value to true explicitly. This removes the need to drop in a bunch of `false` on non-pointables and indicates the symbols are pointable in the specific visitor methods. This also helps having 3 methods with slightly confusing type signatures: (A, B) + (A, B, C) + (A, B, D).